tests/python/test_sparse_deactivate.py

```diff
@@ -9,7 +9,8 @@ def test_pointer():
     n = 16
 
     ptr = ti.root.pointer(ti.i, n)
-    ptr.dense(ti.i, n).place(x)
+    den = ptr.dense(ti.i, n)
+    den.place(x)
```

Comment:
Any good reason to change this?

Reply:
```suggestion
    ptr.dense(ti.i, n).place(x)
```
Was testing, didn't revert, sorry!